### PR TITLE
Update index.js

### PR DESCRIPTION
**Pull request by stephkoltun**
Minor copy edits. Tried to be consistent in using "ml5.js" rather than just "ml5" - but might have missed some!

### CODE REVIEW

File: website/pages/en/index.js [hunk 1]
```diff
@@ -94,15 +94,14 @@ class Why extends React.Component {
         <div className="f1 p5 subtitle getStartedSection">
           <h1> Friendly Machine Learning for the Web. </h1>
           <p className="mt40">
-            ml5 aims to make machine learning accessible to a broad audience of artists,
-            creative coders, and students. Using TensorFlow.js as a foundation, ml5.js
-            provides access to machine learning algorithms and models in the browser
-            with no other external dependencies.
+            ml5.js aims to make machine learning accessible to a broad audience of artists,
+            creative coders, and students. The library provides access to machine learning algorithms and models 
+            in the browser, building on TensorFlow.js with no other external dependencies.
```
Comment (shiffman) [inline website/pages/en/index.js]: I like this rewording! However, I think it's important to be clear that tf.js is a dependency. In other words, this isn't inspired by tf.js but actually a layer "above" it. Maybe it's clear? Or we could say: "building on *top of* tf.js" or "building *a layer above* tf.js"?

**Review by shiffman — COMMENTED** [review]
thank you for these @stephkoltun so great!

File: website/pages/en/index.js [hunk 2]
```diff
@@ -96,7 +96,7 @@ class Why extends React.Component {
           <p className="mt40">
             ml5.js aims to make machine learning accessible to a broad audience of artists,
             creative coders, and students. The library provides access to machine learning algorithms and models 
-            in the browser, building on TensorFlow.js with no other external dependencies.
+            in the browser, building on top of TensorFlow.js with no other external dependencies.
```
Comment (shiffman) [inline website/pages/en/index.js]: Thanks!

**Review by shiffman — APPROVED** [review]

**Comment by matamalaortiz** [comment]
Great, Thanks!
I had some comflicts with this PR so I just added the changes mannually this time. 
Changes should be up in the current version.

**Comment by stephkoltun** [comment]
Okay! I'll close this pull request.

**Pull request closed without merging**